### PR TITLE
Fix tests.

### PR DESCRIPTION
**Pull request by slurms**

### CODE REVIEW

**Comment by slurms** [comment]
@auvipy seems like you just made some similar changes 😂. Could we enable travis for pull requests also? The tests aren't quite fixed yet.

**Comment by slurms** [comment]
Tests should be good to go :+1:.

**Comment by auvipy** [comment]
quite surprized to see travis is not enables for pull requests!

File: actstream/managers.py [hunk 1]
```diff
@@ -190,14 +190,16 @@ def following_qs(self, user, *models, flag=''):
...
         qs = qs.filter(ctype_filters)
 
+        flag = kwargs.get('flag', '')
+
         if flag:
             qs = qs.filter(flag=flag)
         return qs.fetch_generic_relations('follow_object')
 
-    def following(self, user, *models, flag=''):
+    def following(self, user, *models, **kwargs):
```
Comment (zmrenwu) [inline actstream/managers.py]: Which python version raises syntax error? 3.5 or 3.6?

Comment (auvipy) [reply actstream/managers.py]: 3.5

Comment (slurms) [reply actstream/managers.py]: I think even 2.7

**Comment by slurms** [comment]
Thanks @auvipy. Looks like I'll need to add another commit because travis is failing on master still.

**Comment by auvipy** [comment]
yes tox/travis needs adjusted

File: actstream/managers.py (hunk 1 above)
Comment (auvipy) [reply actstream/managers.py]: its not actually python version issue

Comment (zmrenwu) [reply actstream/managers.py]: So what's the problem, I test my fork version in local py3.5 environment and get no errors. The modified codes work same as previous, I think, but a little more verbose.

Comment (slurms) [reply actstream/managers.py]: I thought keyword-only arguments were a syntax error in python 2.7 @auvipy?

```
  File "/home/travis/build/justquick/django-activity-stream/actstream/managers.py", line 180
    def following_qs(self, user, *models, flag=''):
                                             ^
SyntaxError: invalid syntax
```

**Comment by auvipy** [comment]
ERROR: InterpreterNotFound: python3.5
___________________________________ summary ____________________________________
ERROR:   py35-dj2.0-postgres: InterpreterNotFound: python3.5

**Pull request merged**